samples/CommunityToolkit.Maui.Sample/Pages/Views/Expander/ExpanderPageCS.cs

```diff
@@ -40,7 +43,9 @@ public ExpanderPageCS()
 
 						new Label()
 							.Text(".NET Multi-platform App UI (.NET MAUI) is a cross-platform framework for creating mobile and desktop apps with C# and XAML. Using .NET MAUI, you can develop apps that can run on Android, iOS, iPadOS, macOS, and Windows from a single shared codebase.")
-							.Font(italic: true)
+							.Font(italic: true),
+
+						button
```

Comment:
Let's initialize the button in-line here

```suggestion
						new Button()
							.Text("Learn more")
							.Invoke(button => button.Clicked += async (s, e) => await Launcher.OpenAsync("https://dot.net/maui"))
```

Reply:
If we are attempting to show off the `EventToCommandBehavior` feature would it be better to make use of that over the `Clicked` event?